tests/unit/ui/components/TestLoadingText.js

```diff
@@ -0,0 +1,24 @@
+import { shallow } from 'enzyme';
```

Comment:
Oh, it did. This test file was just never renamed after the component changed names (all I did here was rename it).